src/app/Providers/AuthServiceProvider.php

```diff
@@ -16,6 +20,8 @@ class AuthServiceProvider extends ServiceProvider
      */
     protected $policies = [
         'App\Models\Model' => 'App\Policies\ModelPolicy',
+        User::class => AdminPolicy::class,
```

Comment:
Create a separate policy for each model, policy name should follow the mode's name